frontend/ui/widget/buttontable.lua

```diff
@@ -24,6 +24,7 @@ local ButtonTable = FocusManager:new{
     zero_sep = false,
     button_font_face = "cfont",
     button_font_size = 20,
+    auto_focus_first_button = true,
```

Comment:
When should we provide this as `false` ?
I see you provide it when the ButtonTable is the container of bottom buttons. So should we use false when these bottom buttons are not the first focuses thing in the whole widget ? Or is there some other reason/recommandation ?

There are probably other widgets that use ButtonTable for bottom buttons and other things, that you didn't have to visit with this PR for solving your bottom config dialog issue - but they may need some update then ?